src/Post_Command.php

```diff
@@ -748,4 +748,31 @@ private function read_from_file_or_stdin( $arg ) {
 		}
 		return file_get_contents( $readfile );
 	}
+
+	/**
+	 * Get category id if given name/slug
+	 *
+	 * @param string $arg Supplied argument.
+	 * @return array
+	 */
+	private function get_category_ids( $arg ) {
+		$categoires = explode( ',', $arg );
+
+		foreach ( $categoires as $post_category ) {
+			if ( trim( $post_category ) !== '' ) {
+				if ( is_numeric( $post_category ) && (int) $post_category ) {
+					$category_id = category_exists( (int) $post_category );
+				} else {
+					$category_id = category_exists( $post_category );
+				}
+				if ( ! $category_id ) {
+					$category_ids[] = $post_category;
```

Comment:
This `$category_ids[] = $post_category;` can be removed as about to error and not return.